commit 96250267bd7c22b47ca93084112c97e722b18c54
Author: Sune Vuorela <sune@vuorela.dk>
Date:   Fri May 16 18:26:06 2014 +0200

    Make include style be the same for QtCore includes
    
    As a side effect, it also fixes a build for some pieces of software
    that built in the past, but doesn't build now.

diff --git a/kio/kio/global.h b/kio/kio/global.h
index 8a46ae6..7613722 100644
--- a/kio/kio/global.h
+++ b/kio/kio/global.h
@@ -26,7 +26,7 @@
 #include <QtCore/QMap>
 #include <QtCore/QList>
 #include <QtCore/QVariant>
-#include <QFile>  // for QFile::Permissions
+#include <QtCore/QFile>  // for QFile::Permissions
 
 #include <kiconloader.h>
 #include <QtGui/QPixmap> // for pixmapForUrl
